test/cli.js

```diff
@@ -303,6 +305,47 @@ describe('CLI', () => {
         });
     });
 
+    it('starts the inspector with --inspect', (done) => {
+
+        const httpServer = new Http.Server(() => {});
+        httpServer.listen(0, () => {
```

Comment:
This is trying to start the inspector on a random/available port, right? It kinda has a race condition, even though you're unlikely to run into it.